tests/test_habitat_quality.py

```diff
@@ -233,8 +238,8 @@ def test_habitat_quality_regression(self):
                 'deg_sum_f_regression.tif': 33.931896,
                 'quality_c_regression.tif': 7499.983,
                 'quality_f_regression.tif': 4999.9893,
-                'rarity_c_regression.tif': 2500.0000000,
-                'rarity_f_regression.tif': 2500.0000000}.items():
+                'rarity_c_regression.tif': 3333.33335,
+                'rarity_f_regression.tif': 3333.33335}.items():
```

Comment:
I guess it doesn't matter since tests are passing. But I see there's 5 decimals here and 4 on all the subsequent tests where this number appears.

Reply:
This definitely looks like a classic Vim paste mistake. I went for consistency and removed a decimal place. https://github.com/natcap/invest/pull/728/commits/cf5921726822d70ff12f0b95e9d6c27e7d89c821